corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/stylesheets/imports/includes_variables._variables.style.diff.txt

```diff
@@ -16,7 +16,7 @@
 +  xxl: 1160px
 +);
  
--@font-size-base:  12px;
+-@font-size-base:  14px;
```

Comment:
I think this change should also happen in B5. As-is, the font will shrink back down when we migrate.